stable/traefik/templates/configmap.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
 data:
   traefik.toml: |
     # traefik.toml
+    {{ .Values.extraConfig | indent 4 }}
```

Comment:
You probably should not indent this line.

Reply:
fixed